Source/options.cpp

```diff
@@ -171,10 +171,10 @@ void LoadOptions()
 	sgOptions.Graphics.bBlendedTransparancy = getIniBool("Graphics", "Blended Transparency", true);
 	sgOptions.Graphics.nGammaCorrection = getIniInt("Graphics", "Gamma Correction", 100);
 	sgOptions.Graphics.bColorCycling = getIniBool("Graphics", "Color Cycling", true);
-#ifndef USE_SDL1
-	sgOptions.Graphics.bHardwareCursor = getIniBool("Graphics", "Hardware Cursor", false);
-#else
-	sgOptions.Graphics.bHardwareCursor = false;
+#if SDL_VERSION_ATLEAST(2, 0, 0)
+	sgOptions.Graphics.bHardwareCursor = getIniBool("Graphics", "Hardware Cursor", true);
+	sgOptions.Graphics.bHardwareCursorForItems = getIniBool("Graphics", "Hardware Cursor For Items", false);
+	sgOptions.Graphics.nHardwareCursorMaxSize = getIniInt("Graphics", "Hardware Cursor Maximum Size", 128);

```

Comment:
I think just having `Hardware Cursor For Items` is a better idea or it starts to get a bit complicated.

Reply:
These 2 settings are not related. Maximum size applies to the non-item cursor as well.